Source/dom/models/Library.js

```diff
@@ -16,14 +16,20 @@ const AddStatus = {
 
 const LibraryTypeMap = {
   0: 'Internal',
-  1: 'User', // Is this a library added by a user.
-  2: 'Remote', // Is this a library that can be updated using an appcast.
+  1: 'LocalUser',
+  2: 'RemoteUser',
+  3: 'RemoteTeam',
+  4: 'RemoteThirdParty',
 }
 
 const LibraryType = {
   Internal: 'Internal',
-  User: 'User',
-  Remote: 'Remote',
+  User: 'LocalUser', // Mapping `User` to new value `LocalUser`
```

Comment:
Let's make this and `Remote` super obvious that they shouldn't be used going forward. Something like this perhaps? 🤔 

```suggestion
  // @deprecated Please use `LocalUser` instead
  User: 'LocalUser', // This value has been remapped for compatibility and will be removed in the future.
```